### PR TITLE
jabber callback

### PR DESCRIPTION
**Pull request by gadost**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME

lib/ansible/plugins/callback/jabber.py
##### ANSIBLE VERSION

```
ansible 2.1.0.0
  config file = 
  configured module search path = Default w/o overrides
```
##### SUMMARY

Developed callback module send to jabber user playbook info such as start , finish , failed steps.
Finish and failed steps with debug message.

depends:

```
xmpppy
```

env variables:

```
JABBER_SERV  -  jabber server ( f.e.  example.com )
JABBER_TO    -  recipient 
JABBER_USER  -  sender 
JABBER_PASS  -  sender pass
```

### CODE REVIEW

**Comment by mscherer** [comment]
Shouldn't it be xmpp callback ?

File: lib/ansible/plugins/callback/jabber.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                failures = True
+            if s['unreachable'] > 0:
+                unreachable = True
+
+        if failures or unreachable:
+            out = self.debug
+            self.send_msg("%s: Failures detected \n%s \nHost: %s\n Failed at:\n%s" % (name, self.task, h, out))
+        else:
+            out = self.debug
+            self.send_msg("Great! \n Playbook %s complited:\n%s \n Last task debug:\n %s" % (name,s, out))
```
Comment (mscherer) [inline lib/ansible/plugins/callback/jabber.py]: Complited, you mean "completed" ?

**Comment by gadost** [comment]
yes , "completed".
fixed

**Comment by gadost** [comment]
> Shouldn't it be xmpp callback ?

Yes, it is can be

**Pull request merged**